.hemtt/project.toml

```diff
@@ -0,0 +1,31 @@
+name = "Advanced Equipment"
+prefix = "ae3"
+mainprefix = "z"
+
+[properties]
```

Comment:
Are these properties necessary? author seems to be hardcoded everywhere and template ... I don't know. Looks kind of optional to me if this isn't used as a variable that is referenced in some other places I think. How ACE3 does it? They changed to HEMTT v1.0 recently.

Reply:
ACE3 does this the same way.